packages/core/src/components/essentials/tooltip.ts

```diff
@@ -202,7 +209,14 @@ export class Tooltip extends Component {
 	}
 
 	render() {
-		this.tooltip.classed("hidden", true);
+		const isTooltipEnabled = Tools.getProperty(
+			this.model.getOptions(),
+			"tooltip",
+			"enabled"
+		);
+		if (isTooltipEnabled) {
+			this.tooltip.classed("hidden", true);
```

Comment:
so if tooltip isn't enabled we won't hide it?

Reply:
No, we don't need to hide tooltip when tooltip is disabled. When tooltip is disabled, we won't even define tooltip in `init`.